packages/experimental/Badge/src/BadgeColorTokens.win32.ts

```diff
@@ -240,23 +214,22 @@ export const defaultBadgeColorTokens: TokenSettings<BadgeTokens> = (t: Theme) =>
       filled: {
         ...getFilledColorProps(
           {
-            backgroundColor: globalTokens.color.grey[14],
-            backgroundColorDark: globalTokens.color.white,
-            colorDark: t.colors.neutralForeground1,
+            backgroundColor: t.colors.neutralForeground1,
+            color: t.colors.neutralBackground1,
           },
           t,
           getWin32Props,
         ),
       },
       outline: {
-        ...getOutlineColorProps({ color: t.colors.neutralForeground1, colorDark: globalTokens.color.white }, t, getWin32Props),
+        ...getOutlineColorProps({ color: t.colors.neutralForeground3, borderColor: t.colors.neutralStrokeAccessible }, t, getWin32Props),
       },
       tint: {
         ...getTintColorProps(
           {
-            backgroundColor: globalTokens.color.grey[38],
-            color: t.colors.neutralForegroundOnBrand,
-            borderColor: t.colors.transparentBackground,
+            backgroundColor: t.colors.neutralBackground3,
+            color: t.colors.neutralForeground3,
```

Comment:
I had to put here foreground3 because token from the design looks almost the same as background here

Reply:
I thought this was the badge type where the foreground alias token was assigned to backgroundColor and vice versa? Redlines say that the background should neutralForeground3 and color should be neutralBackground1, can you show screenshots of why this doesn't work (other than HC being weird?)

Reply:
I found a bug. There's should be foreground3 instead of background3 for background. That's why backgroundColor wasn't look good. Now it's good:
![image](https://user-images.githubusercontent.com/11574680/185898528-d1f4c792-f9f4-4d3b-b3a5-b4aba5f5addf.png)